lib/app/data/repositories/course_data_repositories.dart

```diff
@@ -10,7 +13,9 @@ List<Course> courses = [
       image: AppImage.pumpkinface,
       duration: '3-10 Min',
       textColor: AppColors.textColor,
-      onTap: () {},
+      onTap: () {
+        Get.to(CourseView(), binding: CourseBinding());
```

Comment:
you do not have to pass the binding on route. 

use named route instead from app-routes.dart

Reply:
done